controllers/movement.py

```diff
@@ -189,6 +190,15 @@ def is_at_goal(self) -> bool:
     def manualdrive(self) -> None:
         if self.debug_trajectory:
             self.generate_trajectory()
+
+        # if we arent trying to move and almost at end of a match
+        if (
+            wpilib.DriverStation.getMatchTime() < 5
```

Comment:
This needs to account for when we're running the robot outside of a match.

Reply:
what is the best way to do this check?

Reply:
Isn't match time documented to be some sort of sentinel value when not in a match or practice mode?

Reply:
If there is I couldn't find it